mlrun/runtimes/base.py

```diff
@@ -224,7 +224,14 @@ def status(self, status):
         self._status = self._verify_dict(status, "status", FunctionStatus)
 
     def _get_k8s(self):
-        return get_k8s_helper()
+        # ideally we are not supposed to access k8s helper through client side
+        # once separated client and server this if statement will be redundant
+        if is_running_as_api():
+            import mlrun.api.utils.singletons.k8s
+
+            return mlrun.api.utils.singletons.k8s.get_k8s_helper()
+
+        return get_k8s_helper(silent=True)
```

Comment:
Maybe explicitly call `mlrun.k8s_utils.get_k8s_helper` to differentiate between the 2 identically named methods?

Reply:
no need for `get_k8s_helper` on client side any more